rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -3136,28 +3082,6 @@ fn test_install_code_rate_limiting_disabled() {
     test.upgrade_canister(canister_id, binary).unwrap();
 }
 
-#[test]
```

Comment:
This test was specifically checking whether an `InstallCodeContext` would fail to be created if compute and memory allocation were out of range. The respective fields and code handling them was removed from the conversion function so the test is no longer valid.